### ISSUE TITLE
Home folder missing when create container from Image saved with commit

### ISSUE THREAD

**Issue by keshavmass0** [danielguerra69/ubuntu-xrdp#42]
Hi Daniel
Thanks for creating this image.
I created a container from this image and it seems to work fine. 
Then I created a new image out of it by Docker commit.
Now when create container from last saved Image /home/ubuntu/ folder is missing due to which We are unable to SSH/RDP.
If you could check this out would be of great help.

**Comment by danielguerra69** [comment]
I can not see what you have done, so check youre own work. What is your  issue with my work ?

**Comment by keshavmass0** [comment]
Daniel you have done a great job. I am just asking your help in below issue. When we normally do below steps with **Ubuntu:xx.xx** images, issue did not seems to appear.

- I created a container from this image and it seems to work fine.
- Then I created a new image out of this container by Docker commit.
- Now when create container from last saved Image /home/ubuntu/ folder is missing due to which We are unable to SSH/RDP.
